src/sentry/api/base.py

```diff
@@ -108,17 +108,16 @@ def create_audit_entry(self, request, **kwargs):
             **kwargs
         )
 
+        extra = {
+            'entry_id': entry.id,
+            'actor_label': entry.actor_label
+        }
         if entry.actor_id:
-            logger.bind(actor_id=entry.actor_id)
+            extra['actor_id'] = entry.actor_id
         if entry.actor_key_id:
-            logger.bind(actor_key_id=entry.actor_key_id)
+            extra['actor_key_id'] = entry.actor_key_id
 
-        logger.info(
-            name='sentry.audit.entry',
```

Comment:
Updated names in https://github.com/getsentry/sentry/pull/3714/commits/58997aa191fa7055a3524bf88e5f5c72c6581dc4.